### PR TITLE
Update apollo3.mdx

### PR DESCRIPTION
**Pull request by lizokm**
Made a few language tweaks to make this page easier to read.

### CODE REVIEW

File: src/platforms/android/configuration/integrations/apollo3.mdx [hunk 1]
```diff
@@ -10,14 +10,16 @@ categories:
 
 <Note>
 
-Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+To be able to capture transactions, you'll need to first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>.
```
Comment (marandaneto) [inline src/platforms/android/configuration/integrations/apollo3.mdx:13]: This sentence is in a bunch of files, mind updating all of them? just search for:

> Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.

**Comment by marandaneto** [comment]
@lizokm mind updating `src/platforms/java/common/performance/instrumentation/apollo3.mdx` as well? it's pretty much a copy-paste expect some technical bits.

File: src/platforms/android/configuration/integrations/apollo3.mdx (hunk 1 above)
Comment (lizokm) [reply src/platforms/android/configuration/integrations/apollo3.mdx:13]: Will do!

**Comment by lizokm** [comment]
@marandaneto updated the Java Apollo3 files to match: https://github.com/getsentry/sentry-docs/pull/7165

**Pull request merged**